server/partials/footer.html

```diff
@@ -21,10 +21,13 @@ <h4 class="ui header">Community</h4>
       <div class="three wide column">
         <h4 class="ui header">Social</h4>
         <div class="ui link list">
-          <a class="item" href="https://twitter.com/fomanticui" target="_blank">
-            <i class="twitter icon"></i>
-            Twitter
-          </a>
+          <a class="item twitter" href="https://twitter.com/fomanticui" target="_blank">
+     <svg width="15" height="15" x="0px" y="-20px" viewBox="0 0 256 256" fill="none" xmlns="http://www.w3.org/2000/svg" style="fill:#000000; vertical-align:middle">
+  <g fill-opacity="0.4" fill="#000000" fill-rule="nonzero" stroke="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke-miterlimit="10" stroke-dasharray="" stroke-dashoffset="0" font-family="none" font-weight="none" font-size="none" text-anchor="none" style="mix-blend-mode: normal">
+    <g transform="scale(10.66667,10.66667)"><path d="M6,3c-1.64497,0 -3,1.35503 -3,3v12c0,1.64497 1.35503,3 3,3h12c1.64497,0 3,-1.35503 3,-3v-12c0,-1.64497 -1.35503,-3 -3,-3zM6,5h12c0.56503,0 1,0.43497 1,1v12c0,0.56503 -0.43497,1 -1,1h-12c-0.56503,0 -1,-0.43497 -1,-1v-12c0,-0.56503 0.43497,-1 1,-1zM6.70117,7l3.94141,5.63281l-3.73633,4.36719h1.33203l3.00195,-3.51562l2.45703,3.51563h3.83984l-4.18555,-5.99023l3.42188,-4.00977h-1.31055l-2.70312,3.16016l-2.20703,-3.16016zM8.56641,7.98828h1.48633l5.63086,8.02344h-1.49609z"></path></g></g>
+  </svg>
+    <span style="vertical-align:middle">Twitter</span>
```

Comment:
That's still not the original brand logo (it does not have a square outline)
Please use the original brand svg and scale it accordingly via css (See suggested CSS Code above)

See differences:
First line is my suggested code using original brand icon from x.com website
Second is your PR 
![image](https://github.com/fomantic/Fomantic-UI-Docs/assets/18379884/32802100-d59c-4771-87e4-256d6a4dae96)

```suggestion
<svg width="1200" height="1227" viewBox="0 0 1200 1227" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path>
</svg> Twitter
```